### DISCUSSION TITLE
How to calculate PG.Q.Value and Lib.Q.Value

### DISCUSSION THREAD

**Discussion by JuWang123** [vdemichev/DiaNN#823] [Q&A]
In my understanding, Q.value is calculated based on the CScore in the report.tsv. I am wondering how DIANN calculate the PG.Q.Value and Lib.Q.Value? Thank you.

**Comment by vdemichev** [comment]
Quite a complex yet unpublished algorithm.

**Reply by JuWang123** [reply]
Thanks for the answer. Is it reasonable to use PG.Q.Value or Global.PG.Q.Value to evaluate the protein group we identified? In addition, it seems Global.PG.Q.Value is always the same for the specific protein group in different runs but PG.Q.Value not. Therefore, I think Global.PG.Q.Value is a better one to evaluate that. What's your thought about it?

**Reply by vdemichev** [reply]
I would suggest to follow the guidance in DIA-NN docs, e.g. when to use 'Lib' vs 'Global' filtering.
When to use the run-specific one - depends on whether you feel you need it for your downstream analysis.

**Comment by YuL-BI** [comment]
Hi Vadim,

In the DIA-NN analysis setting, what does the Maximum number of variable modifications refer?  Types (2 Ox(M) and 3 Phospho = 2) or all together the numbers of various modifications (2 Ox(M) and 3 Phospho = 5)?

Thank you.

Lu
